h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1161,6 +1161,7 @@ public void setZValues(double [] zValues, double dispersion, boolean dispersionE
   public static class GLMOutput extends Model.Output {
     Submodel[] _submodels = new Submodel[0];
     DataInfo _dinfo;
+    double[] _ymu;
```

Comment:
I believe this information is redundant - this seems to be the same value as "prior class distribution"

<img width="1106" alt="Screen Shot 2021-11-24 at 10 30 55 AM" src="https://user-images.githubusercontent.com/20690218/143267933-cc1c2fc4-50da-451b-a23b-57e57980a9c9.png">

maybe we should expose that instead?

Reply:
there is a difference - for regression, this value holds the mean response

Reply:
@michalk:  @marek needs the two parameters to generate a metric class as follows: 

  public GLMMetricBuilder(String[] domain, double [] ymu, GLMWeightsFun glmf, int rank, boolean computeMetrics, boolean intercept, MultinomialAucType aucType)

That is why he is looking for ymu and rank variable.

You are right that the ymu and the priorClassDist.  They are almost the same except in the case of regression.  That is why I choose to use ymu.  Is this okay?